src/db/create_tables.py

```diff
@@ -0,0 +1,58 @@
+# db/create_tables.py
+# Description: This file contains the functions to create the tables in the database.
+
+from db.util import load_sql_file
+
+def create_consumption_schema(logger, conn):
+    create_table_products(logger, conn)
+    create_table_holdings_btc(logger, conn)
+    create_table_holdings_btc_bfill(logger, conn)
+    create_table_inflows_btc(logger, conn)
+    create_table_inflows_btc_bfill(logger, conn)
+    create_table_inflows_btc_bfill_inline(logger, conn)
+    create_table_inflows_btc_bxfill(logger, conn)
+    create_table_inflows_btc_sma5(logger, conn)
+
+
+def create_table_products(logger, conn):
+    PRODUCTS_CREATE = load_sql_file("db/queries/products_create.sql")
```

Comment:
I like this idea of having all sql queries in a dedicated folder, but I would not hard code its path here, but rather define the path of the folder in the config file or as a constant and then each sql file path will simply be obtained by concatenating the folder path with the file name